DigitalLearningSolutions.Web/ViewModels/Frameworks/CompetencyResourceSignpostingViewModel.cs

```diff
@@ -1,26 +1,36 @@
-﻿using System;
+﻿using DigitalLearningSolutions.Web.Helpers;
+using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage;
+using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace DigitalLearningSolutions.Web.ViewModels.Frameworks
 {
-    public class CompetencyResourceSignpostingViewModel
+    public class CompetencyResourceSignpostingViewModel : BaseSearchablePageViewModel
```

Comment:
We don't need to implement the BaseSearchablePageViewModel for this search interface. We will be relying on the external LH OpenAPI search endpoint to retrieve search results rather than using an internal search mechanism. We don't need much back-end work at all for this ticket, it is just setting up the View and the Controller. DLSV2-440 will call the API end point and retrieve results.